doc/build/sysbuild/index.rst

```diff
@@ -0,0 +1,370 @@
+.. _sysbuild:
+
+Sysbuild (System build)
+#######################
+
+Sysbuild is a higher level CMake and Kconfig based build system which support
+configuring and building a Zephyr application together with additional projects,
+both Zephyr based but also non-Zephyr based projects.
+
+A common use-case is building a Zephyr application together with MCUboot.
+
+Architectural Overview
+**********************
+
+Architectural overview of the sysbuild project.
```

Comment:
Is this line needed?

Reply:
Done, addressed by https://github.com/zephyrproject-rtos/zephyr/pull/43846#pullrequestreview-1060453559